app/index.js

```diff
@@ -711,16 +775,14 @@ module.exports = Generator.extend({
         { name: 'autoscalingName', message: 'Enter name (blank for default):' }
       ]
       return this.prompt(prompts).then((answers) => {
-        this.services.autoscaling[0].name = answers.autoscalingName || this.services.autoscaling[0].name
+        this.services.autoscaling.serviceInfo.name = answers.autoscalingName || this.services.autoscaling.serviceInfo.name
       })
     },
 
     promptConfigureWatsonConversation: function () {
       if (this.skipPrompting) return
       if (!this.servicesToConfigure) return
-      if (!this.servicesToConfigure.watsonconversation) return
-
-      this.log()
```

Comment:
please add this log back for consistency of presentation

Reply:
done